sag_desc.par

```diff
@@ -81,30 +81,30 @@ Artemis' half arrow
 7:11-comma (45927/45056)
 Demeter's scythe
 /|~
-�
-5:23 (or 245 or 7:13) -S-diesis (46/45)
+
+5:23-S-diesis (46/45)
 Hermes' winged caduceus
 (|(
-q
-5:11 (or 625) -S-diesis (45/44)
+g
+5:11-S-diesis (45/44)
 Aphrodite's perfect curves
 //|
 =
 25-S-diesis, minor diesis (6561/6400)
 Amphitrite's crab-claw hairclip
 /|)
-n
-35 (or 125 or 13) -M-diesis (8505/8192)
+%
+35-M-diesis (36/35)

```

Comment:
switched the 35M and the 35L

Reply:
Well spotted. But I think we should continue to include secondaries where they are also given in the SMuFL documentation. However I'd list them in order of increasing SoPF>3, which is, I think, how they are listed in SMuFL (against the upward symbol only).

Maybe use a tilde to mean approximate, before the secondaries, as in SMuFL.

Reply:
Will do.